parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -677,6 +681,17 @@ parameter_types! {
 	// TODO: placeholder value - choose a real one
 	pub const MaxUpgradeDataSize: u32 = 1024;
 	pub const RelayNetwork: NetworkId = Rococo;
+	pub const ControlPalletId: PalletId = PalletId(*b"snow/ctl");
+
+	// Multiplier based on gas report as follows,assume gas cost for a no-op operation is 1000
+	// | createAgent                                     | 839             | 184709 | 237187 | 237187 | 9       |
+	// | createChannel                                   | 399             | 31023  | 2829   | 75402  | 5       |
+	// | updateChannel                                   | 817             | 15121  | 3552   | 36762  | 5		 |
+	// | transferNativeFromAgent                         | 770             | 21730  | 21730  | 42691  | 2       |
+	pub const CreateAgentMultiplier: u128 = 237;
+	pub const CreateChannelMultiplier: u128 = 75;
+	pub const UpdateChannelMultiplier: u128 = 37;
+	pub const TransferNativeFromAgentMultiplier: u128 = 43;
```

Comment:
This multipliers look unused.

Reply:
https://github.com/Snowfork/cumulus/pull/63/commits/1a1ab19a6e0d4449211ee24897ee7211b72cf9a1